src/core/reducers/addons.js

```diff
@@ -26,6 +26,7 @@ export default function addon(state = initialState, action) {
       } else if (thisAddon.current_version && thisAddon.current_version.files.length > 0) {
         newState[key] = {
           ...thisAddon,
+          iconUrl: thisAddon.icon_url,
```

Comment:
This is quite unforunate but I looked into switching everything over to camel cased and it looks like a good amount of work. I filed #1536.